apps/are_form/fields/first-page.js

```diff
@@ -17,20 +23,27 @@ module.exports = {
     }]
   },
   'appeal-stage': {
-    validate: ['required'],
-    className: ['typeahead', 'js-hidden'],
-    options: [' '].concat(staticAppealStages.getstaticAppealStages())
+    mixin: 'select',
+    validate: 'required',
+    legend: {
+      className: 'visuallyhidden'
+    },
+    options: [{
+      value: '',
+      label: 'fields.appeal-stage.options.null'
+    }].concat(staticAppealStages.getstaticAppealStages())
   },
-
-  'start-date': {
+  'start-date': dateComponent('start-date', {
+    labelClassName: 'visuallyhidden',
     validate: ['required', 'date'],
-    legend: 'fields.start-date.legend',
+    legend: {
+      className: 'visuallyhidden'
+    },
     hint: 'fields.start-date.hint'
-  },
+  }),
   'start-date-day': {
     validate: ['required', 'numeric']
   },
-
   'start-date-month': {
     validate: ['required', 'numeric']
   },
```

Comment:
is there a reason for this to be in a separate file for this? could it not just be in `index.js`